src/main/java/com/sap/oss/phosphor/fosstars/model/value/VersionRange.java

```diff
@@ -10,42 +11,61 @@
  * The class holds a start and end version.
  */
 @JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonAutoDetect(fieldVisibility = Visibility.ANY)
 public class VersionRange {
 
   /**
    * First vulnerable version (including).
    */
-  private final String versionStart;
+  private final String versionStartIncluding;
 
   /**
    * Last vulnerable version (including).
    */
-  private final String versionEnd;
+  private final String versionEndIncluding;
 
   /**
-   * Create a version range with start and end version.
-   * Both versions are including.
-   * Start and end version may be null.
+   * First vulnerable version (excluding).
+   */
+  private final String versionStartExcluding;
+
+  /**
+   * Last vulnerable version (excluding).
+   */
+  private final String versionEndExcluding;
```

Comment:
I am not sure if we need these fields (please note that this class doesn't model data from NVD -- we have classes in the `nvd` package that do that). Plus, the class doesn't check that all these four fields are correct (overlaps, etc). If you still would like to make this class aware of open/closed ends, I would suggest to use flags like in https://github.com/SAP/fosstars-rating-core/blob/master/src/main/java/com/sap/oss/phosphor/fosstars/model/math/DoubleInterval.java